mksusecd

```diff
@@ -616,7 +622,7 @@ if($opt_create || $opt_list_repos) {
     }
   }
 }
-
+print "All Done";
```

Comment:
Over my dead body.

Reply:
Sorry, this was only for debuging where th output of mksusecd begins and mine ends....
Forgot to remove it.

Reply:
Ok. I'm basically fine with the changes, just the few comments.